backend/sublet/migrations/0003_alter_offer_phone_number.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.22 on 2023-10-29 17:24
+
+import phonenumber_field.modelfields
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
nit: let's delete all these migration files and re-makemigrations to put all of these changes into one migration file